### PR TITLE
Style radio buttons to better fit in with other form controls

### PR DESCRIPTION
**Pull request by Neloop**
image

### CODE REVIEW

**Comment by Neloop** [comment]
Standalone radio inputs resolved...

File: src/components/forms/Fields/StandaloneRadioField.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+const StandaloneRadioField = ({ name, value }) => {
+  return (
+    <div className={'radio-container'}>
```
Comment (krulis-martin) [inline src/components/forms/Fields/StandaloneRadioField.js]: className={'radio-container'} -> className="radio-container"

File: src/components/forms/Fields/StandaloneRadioField.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <label>
+        <Field name={name} component="input" type="radio" value={value} />
+        <span className={'radiomark'}></span>
```
Comment (krulis-martin) [inline src/components/forms/Fields/StandaloneRadioField.js]: -"-

File: src/components/forms/Fields/RadioField.js [hunk 3]
```diff
@@ -8,12 +8,14 @@ const RadioField = ({ input, meta: { error, warning }, options }) => {
     <FormGroup validationState={error ? 'error' : warning ? 'warning' : undefined} controlId={input.name}>
       {options.map(({ key, name }, idx) => (
         <Radio
+          className={'radio-container'}
```
Comment (krulis-martin) [inline src/components/forms/Fields/RadioField.js]: -"-

File: src/components/forms/Fields/RadioField.js [hunk 4]
```diff
@@ -8,12 +8,14 @@ const RadioField = ({ input, meta: { error, warning }, options }) => {
...
           key={`radio${idx}-${key}`}
           name={input.name}
           value={key}
           checked={input.value === key}
           onChange={input.onChange}>
           {name}
+          <span className={'radiomark'}></span>
```
Comment (krulis-martin) [inline src/components/forms/Fields/RadioField.js]: -"-

File: src/components/forms/StandaloneRadioInput/StandaloneRadioInput.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      <label>
+        <input
+          type="radio"
+          name={name}
+          value={value}
+          checked={checked}
+          disabled={disabled}
+          readOnly={readOnly}
+        />
+        <span className={'radiomark'}></span>
```
Comment (krulis-martin) [inline src/components/forms/StandaloneRadioInput/StandaloneRadioInput.js]: -"-

File: src/components/forms/StandaloneRadioInput/StandaloneRadioInput.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StandaloneRadioInput = ({ name, value, checked, disabled, readOnly }) => {
+  return (
+    <div className={'radio-container'}>
```
Comment (krulis-martin) [inline src/components/forms/StandaloneRadioInput/StandaloneRadioInput.js]: -"-

**Pull request merged**